src/Universum/Container/Class.hs

```diff
@@ -114,7 +101,12 @@ type instance Element IS.IntSet = Int
 -- @'null' ≡ 'List.null' . 'toList'@
 --
 class ToList t where
-    {-# MINIMAL toList #-}
+    -- | Type of element for some container. Implemented as a type family because
+    -- some containers are monomorphic over element type (like 'T.Text', 'IS.IntSet', etc.)
+    -- so we can't implement nice interface using old higher-kinded types approach.
+    type Element t :: *
```

Comment:
Maybe add explanation into docs why it has to be inside `ToList` class?

Reply:
I added some explanation. But I don't know how to explain benefits because it's hard to explain...